js/src/product-feed/review-request/index.js

```diff
@@ -15,7 +15,7 @@ import REVIEW_STATUSES from './review-request-statuses';
 import './index.scss';
 
 const showNotice = ( status ) =>
-	Object.keys( REVIEW_STATUSES ).includes( status );
+	Object.keys( REVIEW_STATUSES ).includes( status ) && status !== 'APPROVED';
```

Comment:
❔ According to the visual design, looks like the notice is not shown when the status is `UNDER_REVIEW`?

Reply:
Right! Forgot to implement this. Thanks for the catch ❤️ 

Implemented here: 

https://github.com/woocommerce/google-listings-and-ads/pull/1425/commits/352930e073726fbecf77bad300c73d21ee96e597